### ISSUE TITLE
[Bug]: page-orientation doesn't work with print()

### ISSUE THREAD

**Issue by adatta02** [puppeteer/puppeteer#8359]
### Bug description

I'm trying to use the page-orientation CSS declaration to print a PDF with some pages rotated clockwise or counter-clockwise.

This example HTML from Chromium works correctly when I print to PDF in Chrome/Chromium, https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/web_tests/printing/page-orientation-propagated.html

I uploaded that HTML to https://setfive-public.s3.amazonaws.com/page-orientation-propagated.htm

But when I try to print this page to PDF via Puppeteer the page-orientation rule is ignored and the pages are all oriented normally. Even stranger, if I launch Puppeteer with headless=false and manually CRTL+P with the page open the print preview screen correctly reflects the clockwise/counter clockwise rotation.

You can reproduce by running this nodejs script. Both PDFs will have all of their pages oriented normally:

```
const puppeteer = require("puppeteer");

(async () => {
    
console.log('awaiting launch...');
    
    const browser = await puppeteer.launch({
        headless: true,
        defaultViewport: null,
        args: ["--start-fullscreen"],        
    });

    console.log('new page...');
    
    const page = await browser.newPage();
    await page.setViewport({
        width: 1920,
        height: 1080,
    });


    try {
        await page.goto('https://setfive-public.s3.amazonaws.com/page-orientation-propagated.html', {waitUntil: 'domcontentloaded'});
    }catch(err) {
        console.error(err);
        process.exit(0);
    }

    await page.pdf({
        path: 'preferOutput.pdf',
        printBackground: true,
        preferCSSPageSize: true,                
        margin: {
            top: ".15in",
            right: ".15in",
            left: ".15in",
            bottom: ".15in",
        },
    });

    await page.pdf({
        path: 'output.pdf',
        printBackground: true,        
        margin: {
            top: ".15in",
            right: ".15in",
            left: ".15in",
            bottom: ".15in",
        },
    });   
    
    process.exit(0); 
})();
```

And then you can see if you launch with headless=false and try to print from within Puppeteer using CRTL+P it is correctly interpreting the page-orientation rule there:

![Screenshot from 2022-05-17 14-25-12](https://user-images.githubusercontent.com/51879/168884361-7b84c457-0f49-4861-b58f-3cb6df164409.png)

### Puppeteer version

14.1.0

### Node.js version

v16.14.1

### npm version

8.5.0

### What operating system are you seeing the problem on?

Linux

**Comment by ShehanAT** [comment]
I am able to replicate this issue on my end. 
So if I'm not mistaken, you want the PDF pages to have the page orientation CSS applied to them right?

**Comment by adatta02** [comment]
@ShehanAT yup that's right. Whoops just seeing the S3 link from above is broken.

https://setfive-public.s3.amazonaws.com/page-orientation-propagated.html is a working link.

And https://setfive-public.s3.amazonaws.com/page-orientation-propagated.pdf is the output of doing a "Print to PDF" in Chrome and what I'm expecting from Puppeteer.

Thanks!

**Comment by maltoe** [comment]
FYI: I'm facing the same issue without Puppeteer (instead directly talking to the remote debugging port in chromium) - when printing a page with `page-orientation` through the print dialog, all pages will have the correct orientation and rotation, when printed with [printToPDF](https://chromedevtools.github.io/devtools-protocol/tot/Page/#method-printToPDF) it fails. MDN contains another example in their [article on `page-orientation`](https://developer.mozilla.org/en-US/docs/Web/CSS/@page/page-orientation). Second page should be portrait but with content rotated to the left.

**Comment by OrKoN** [comment]
I filed https://issues.chromium.org/issues/332914943

**Comment by OrKoN** [comment]
It's fixed (should be released with M126)

**Comment by OrKoN** [comment]
This will be fixed once 22.11.0 is released https://github.com/puppeteer/puppeteer/pull/12573
